<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易Python库安装器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="std_output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Mirror">
       <property name="text">
        <string>镜像源</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="MirrorChooser">
       <item>
        <property name="text">
         <string>阿里云</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>清华大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>网易</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>腾讯</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>东软</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>华为</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PackageName">
       <property name="text">
        <string>包名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="PackageNameInput">
       <property name="placeholderText">
        <string>输入包名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DownloadButton">
       <property name="text">
        <string>下载</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Powered By Elin@2021</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>安装</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>20</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="libcheck">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>检查</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="std_output_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAuthor"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAuthor">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
